app/src/components/account/UserMenu.vue

```diff
@@ -43,12 +47,18 @@
             />
             <div>
               <span class="address">{{ address.address | formatAddress }}</span>
+              <span class="address-network">{{ address.networkId }}</span>
```

Comment:
i saw your comments in slack but i don't think IDs is the right solution. maybe network titles (why IDs?!) - but maybe a little testnet tag would work too.

Reply:
We could add that cool purple test icon Charlie made. I should have it somewhere... or maybe lost it :sweat_smile: